cli/mender-artifact/copy_test.go

```diff
@@ -111,18 +109,25 @@ func TestCopyRootfsImage(t *testing.T) {
 		},
 		{
 			name: "error: please enter a path into the image",
-			argv: []string{"mender-artifact", "cp", "foo", "menderimg.mender:"},
+			initfunc: func(ipath string) {
+				f, err := os.Create("foo.txt")
+				f.Close()
+				require.Nil(t, err)
+			},
+			argv: []string{"mender-artifact", "cp", "foo.txt", "menderimg.mender:"},
 			err:  "please enter a path into the image",
+			verifyTestFunc: func(ipath string) {
+				os.Remove("foo.txt")
+			},
 		},
 		{
 			name:     "write artifact_info file to stdout",
-			argv:     []string{"mender-artifact", "cat", ":/etc/mender/artifact_info"},
+			argv:     []string{"mender-artifact", "cat", "<artifact|sdimg|fat-sdimg>:/etc/mender/artifact_info"},
```

Comment:
I can see what you are doing.

In the one hand it increases the complexity of the test execution below, but in the other hand it improve readability in the command line arguments here in the test cases definitions. Nice work!